code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -73,12 +74,9 @@
 		removed(imp_in)
 	return ..()
 
-
 /obj/item/weapon/implant/proc/get_data()
 	return "No information available"
 
 /obj/item/weapon/implant/dropped(mob/user)
 	..()
 	. = 1

```

Comment:
since qdel() is called in the ..() shouldn't it be safer to have . = 1 before the ..() ?